tests/time_tests/test_runner/conftest.py

```diff
@@ -59,6 +59,18 @@ def pytest_addoption(parser):
         help="Number of iterations to run executable and aggregate results",
         default=3
     )
+    test_args_parser.addoption(
+        "--input_precision",
+        type=str,
+        help="Change input model precision",
+        default=""
+    )
+    test_args_parser.addoption(
+        "--output_precision",
```

Comment:
This information should be defined per model, so need to parse it from test config (like `test_conf.yml`). Otherwise it will affect all the models from test run